lib/features/sign_up/view_model/sign_up_bloc.dart

```diff
@@ -66,11 +66,10 @@ class SignUpBloc extends Bloc<SignUpEvent, SignUpState> {
       displayName = HatSpaceStrings.current
           .defaultUserDisplayName(DateTime.now().toUtc());
       await _authenticationService.updateUserDisplayName(displayName);
-
-      // save display name and avatar
-      await _storageService.member
-          .saveNameAndAvatar(userDetail.uid, displayName, userDetail.avatar);
     }
+    // save display name and avatar
```

Comment:
Do not move this out of if statement, we only update the user's displacement if their display name is empty